db/schema.rb

```diff
@@ -79,7 +79,7 @@
     t.string "email"
     t.integer "call_for_evidence_response_form_id"
     t.text "postal_address"
-    t.index ["call_for_evidence_response_form_id"], name: "index_cfes_participations_on_cfes_response_form_id"
+    t.index ["call_for_evidence_response_form_id"], name: "index_cons_participations_on_cons_response_form_id"
```

Comment:
This shouldn't be here. Weird how this keeps regenrating itself in some people's local envs

Reply:
I think this happens if people only ever run migrations and don't do DB resets every so often. (If  this existed once in a branch but never got merged, it'll be in the schema of the actual DB of people who worked on that branch and ran the migration that included it, and the way db/schema is made, I believe, is just dumping the actual DB schema at the end of the migration. If you get the current main branch and then do a db:reset that should fix it from appearing again?)